certified-connectors/CiviQ/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2868 @@
+                  "type": "string",
+                  "description": "callback-url",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal",
+                  "title": ""
+                }
+              },
+              "required": [
+                "callback-url"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/files/{FileUUID}.json": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "uuid": {
+                  "type": "string",
+                  "description": "uuid"
+                },
+                "name": {
+                  "type": "string",
+                  "description": "name"
+                },
+                "mime": {
+                  "type": "string",
+                  "description": "mime"
+                },
+                "size": {
+                  "type": "string",
+                  "description": "size"
+                },
+                "attachment_uri": {
+                  "type": "string",
+                  "description": "attachment_uri"
+                },
+                "url": {
+                  "type": "string",
+                  "description": "url"
+                },
+                "content": {
+                  "type": "string",
+                  "description": "content"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "schema": {
+              "type": "string"
+            }
+          },
+          "403": {
+            "description": "Forbidden",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            }
+          },
+          "404": {
+            "description": "Not found",
+            "schema": {
+              "type": "string",
+              "description": "Could not find resource"
+            }
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "schema": {
+              "type": "string"
+            }
+          }
+        },
+        "summary": "Get File",
+        "description": "Operation to retrieve a specific file from the portal.",
+        "operationId": "Get-File",
+        "tags": [
+          "Files"
+        ],
+        "parameters": [
+          {
+            "name": "FileUUID",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "File UUID",
+            "description": "File UUID",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      }
+    },
+    "/surveys/{SurveyID}/schema.json": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
```

Comment:
@RicardoMartins1993 empty schema is not allowed. please define a proper schema. Please fix in all the swagger file.

Reply:
@Amjed-Ayoub  Done